### ISSUE TITLE
Question on localization - this plugin specifies default locale as Japanese

### ISSUE THREAD

**Issue by battika** [mapsplugin/cordova-plugin-googlemaps#2116]
**I'm submitting a ...**  (check one with "x")
- [X] question
- [ ] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [X] **Android**
- [X] **iOS**

**cordova information:**  (run `$> cordova plugin list`) 

```
com.googlemaps.ios 2.6.0 "Google Maps SDK for iOS"
cordova-plugin-camera 4.0.2 "Camera"
cordova-plugin-googlemaps 2.2.5 "cordova-plugin-googlemaps"
```

**If you use `@ionic-native/google-maps`, please tell the package.json (only `@ionic-native/core` and `@ionic-native/google-maps` are fine mostly)**
```
    "@ionic-native/camera": "4.5.2",
    "@ionic-native/core": "4.5.2",
    "@ionic-native/google-maps": "4.5.2",

```

**Request for information**
One of the latest updates to this plugin introduced the following lines in plugin.xml:
```
        <config-file target="*-Info.plist" parent="CFBundleDevelopmentRegion">
          <array>
            <string>Japanese</string>
          </array>
        </config-file>
        <config-file target="*-Info.plist" parent="CFBundleLocalizations">
            <array>
                <string>ja</string>
            </array>
        </config-file>
```
It seems it is causing the app to think it is localized in Japanese, so my native error messages as well as cordova-plugin-camera controls are displayed in Japanese language. Before Google Maps plugin version 2.2.5 they were all displayed in English. I am using Ionic and wondering if there is an easy way to override these settings other than creating a fork of this plugin and change the corresponding lines in the plugin.xml file? 

Thank you in advance.

**Comment by wf9a5m75** [comment]
Use the multiple_maps branch.

**Comment by battika** [comment]
I just did and now I have this issue

https://github.com/mapsplugin/cordova-plugin-googlemaps/issues/2104

**Comment by metz0r** [comment]
I also have this issue. I tried setting defaultlocale in config.xml but didn't help. My app is now in japanese. Are you planning on fixing this in 2.2.5 @wf9a5m75  ?

**Comment by battika** [comment]
It will be fixed in 2.2.6.
